<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bedside Monitoring GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_port">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_patient">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>511</width>
      <height>192</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Patient ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Heart BPM</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>In Bed</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="alert_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ALERT:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="alert_notif">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ack">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Active Devices</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Urgent Patients</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nt">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>500</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>See Network Topology</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ack_done">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_patient">
       <property name="text">
        <string>Patient:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_patient_count">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>378</x>
      <y>90</y>
      <width>75</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_routing">
       <property name="text">
        <string>Routing:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_routing_count">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>80</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_gateway">
       <property name="text">
        <string>Gateway:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_gateway_count">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
